GOVERNANCE.md

```diff
@@ -0,0 +1,101 @@
+
+## Roles and Responsibilities
+
+### Community members
+_Everyone_ who is involved in any form with the project must abide by the project’s [Code of Conduct](). Everyone is expected to be respectful of fellow community members and to work collaboratively respective of the Code of Conduct. Consequences for not adhering to these Guidelines are listed in their respective documents.
+
+### Users
+Users are community members who have a need for the project. They are typically developers building WordPress themes. Anyone can be a User; there are no special requirements and WP Rig is licensed under [GPL v3.0](https://github.com/wprig/wprig/blob/master/LICENSE). Common User contributions include using WP Rig to build themes, evangelizing the project (e.g., display a link on a website and raise awareness through word-of-mouth), informing developers of strengths and weaknesses from a new user perspective, or providing moral support (a “thank you” goes a long way).
+
+Users who continue to engage with the project and its community will often become more and more involved. Such Users may find themselves becoming [Contributors](#Contributors), as described in the next section.
+
+### Contributors
+Contributors are community members who contribute in concrete ways to the project, most often in the form of code and/or documentation submitted through PRs. Anyone can become a Contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements, and no selection process. Contributors must follow the [Code of Conduct]().
+
+Contributors:
+- Have read-only access to source code and therefore can submit changes via pull requests.
+- Have their contribution reviewed and merged by a [Paintainer](#Maintainers) or [Owner](#Owners). Owners and Maintainers work with Contributors to review their code and prepare it for merging.
+- May also review pull requests. This can be helpful, but their approval or disapproval is not decisive for merging or not merging PRs.
+
+As Contributors gain experience and familiarity with the project, their profile within, and commitment to, the community will increase. At some stage, they may find themselves being nominated for becoming a Maintainer by an existing Maintainer or Owner.
+
+### Maintainers
+Maintainers are community members who have shown that they are committed to the continued development of the project through ongoing engagement with the community. Maintainers are given push/write access to the project’s GitHub repos.
+
+Maintainers:
+
+- Are expected to work on public branches of their forks and submit pull requests to the `development` branch.
```

Comment:
This is in direct conflict with `Maintainers are given push/write access to the project’s GitHub repos`. Someone with write access does not need to maintain/submit PRs from a fork.

Reply:
I dont necessarily agree. Just because you have push/write access doesnt mean you shouldnt follow the process of going through pull requests BUT perhaps the point can be made clearer.

Reply:
This is a governance document. It lays out the rules we choose to follow, not what is technically possible. Does that make sense?